### PR TITLE
bug 842340 - remove watch_menu waffle flag

### PR DESCRIPTION
**Pull request by groovecoder**
Please also test watch/subscription notifications, and verify I didn't break helpfulness notifications with my change.

### CODE REVIEW

File: kuma/static/js/components.js [hunk 1]
```diff
@@ -559,6 +564,8 @@
                 return $notifications;
             },
             growl: function(message, options) {
+                var updateFunc = updateMessageHTML
```
Comment (stephaniehobson) [inline kuma/static/js/components.js]: Missing semi colon.

File: kuma/static/js/components.js [hunk 2]
```diff
@@ -567,6 +574,9 @@
                 // Merge options with defaults
                 options = $.extend({}, defaults, options || {});
 
+                if (options.type == 'text') {
+                    updateFunc = updateMessageText
```
Comment (stephaniehobson) [inline kuma/static/js/components.js]: Missing semi colon.

Comment (stephaniehobson) [inline kuma/static/js/components.js]: It's like you normally code in Python or something.

**Comment by stephaniehobson** [comment]
:+1:  with semi colons.

File: kuma/static/js/components.js (hunk 2 above)
Comment (groovecoder) [inline kuma/static/js/components.js]: :tongue:

Comment (groovecoder) [inline kuma/static/js/components.js]: CoffeeScript

**Comment by groovecoder** [comment]
Updated. Even ran `jshint` on it for the lulz.

**Comment by stephaniehobson** [comment]
:+1:

**Pull request merged**